### PR TITLE
Issue 48 chrome jank

### PR DESCRIPTION
**Pull request by Roger-Heathcote**
Fixed viewport height on mobile for all but StoryPage.

### CODE REVIEW

**Comment by Roger-Heathcote** [comment]
Oops, Kat's got a PR up that might clash, gonna take this one down til its merged.

**Comment by Roger-Heathcote** [comment]
Okay, Kat's changes merged now.

File: src/pages/ActionsPage.js [hunk 1]
```diff
@@ -23,44 +24,47 @@ function ActionsPage() {
...
-      </MyGrid>
+      <Div100vh>
+        <MyGrid
+          container
+          direction="column"
+          alignContent="stretch"
+          alignItems="center"
+          justify="space-between"
+        >
+          <Grid item>&nbsp;</Grid>
```
Comment (Alexreid95) [inline src/pages/ActionsPage.js:35]: is there a reason why &nbsp; is there?

Comment (akomiqaia) [reply src/pages/ActionsPage.js:35]: uuuu i didn't know you could comment here

**Pull request closed without merging**